Handle Upload Section validations

alaasaedy opened the pull request:
- Change validations messages positions inside Upload Section component.
- Make sure its logic work as planned.

alaasaedy committed:
Change validations mesaages positions inside upload section component and make sure its logic work as needed.

alaasaedy committed:
Merge branch 'develop' into handle-uploadsection-validation

alaasaedy committed:
Change validations mesaages positions inside upload section component and make sure its logic work as needed.

alaasaedy committed:
Merge branch 'handle-uploadsection-validation' of https://github.com/m3ntorship/pickly.io into handle-uploadsection-validation

MAlngaawy reviewed (changes requested):
it seems you discard the changes that happened in the upload route when you fix the conflict,
please re-add the missing upload route syles from the develop branch.

MAlngaawy reviewed (changes requested):
It should look like that 
![Screenshot from 2020-11-02 11-25-55](https://user-images.githubusercontent.com/46047009/97851763-78452380-1cfe-11eb-96b8-80385bf96057.png)

but when you discard the income changes from the develop branch that's what happened
![Screenshot from 2020-11-02 11-29-18](https://user-images.githubusercontent.com/46047009/97851903-ab87b280-1cfe-11eb-94e7-32657767cb92.png)
 
please, fix this style issue

alaasaedy committed:
Merge branch 'develop' of https://github.com/m3ntorship/pickly.io into handle-uploadsection-validation

alaasaedy committed:
fix styling issue inside Input Field component

alaasaedy committed:
remove Post Something Heading,
Add container to wrap whole section,
Change Input Field padding to match design

alaasaedy committed:
change input field padding

MAlngaawy committed:
edit some styles for input fields

MAlngaawy reviewed (approved):
LGTM
